DemoApp/LoginViewController.swift

```diff
@@ -75,19 +75,30 @@ class LoginViewController: UIViewController {
 
 extension LoginViewController: UITableViewDelegate, UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        builtInUsers.count + 1 // +1 for the last static cell
+        builtInUsers.count + 3 // +1 for the last static cell
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath) as! UserCredentialsCell
         
         if indexPath.row == builtInUsers.count {
-            // Last cell
+            // Anonymous user
+            cell.nameLabel.text = "Anonymous user"
+            cell.descriptionLabel.text = ""
+            cell.avatarView.image = UIImage(systemName: "person")
+            cell.avatarView.backgroundColor = .clear
+        } else if indexPath.row == builtInUsers.count + 1 {
+            // Guest user
+            cell.nameLabel.text = "Guest user"
+            cell.descriptionLabel.text = "user id: guest"
+            cell.avatarView.image = UIImage(systemName: "person.fill")
+            cell.avatarView.backgroundColor = .clear
+        } else if indexPath.row == builtInUsers.count + 2 {
+            // Advanced options
             cell.nameLabel.text = "Advanced Options"
             cell.descriptionLabel.text = "Custom settings"
             cell.avatarView.image = UIImage(named: "advanced_settings")
-            cell.avatarView.backgroundColor = .systemGray
-            
+            cell.avatarView.backgroundColor = .clear
```

Comment:
We probably can leverage `DemoUserType` and have the data of this table view be:

`var users: [DemoUserType]`

And maybe introduce a new type `DemoUserType.custom(...)`

With this, all the data of the table is derived from this property, and we don't need hard coded counts etc...

Reply:
Did you know that "Advanced Options" didn't work at all? Like, the button is not wired 😂 I didn't know!
I'll switch to using enums and remove "Advanced Options" for now

Reply:
We already have that issue reported: https://stream-io.atlassian.net/browse/CIS-1633 https://github.com/GetStream/stream-chat-swift/issues/1830

Reply:
Not sure if it is a good idea to remove that. Is that much work to wire that button? Or leave it non wired then 🤔